public/js/controllers/aside-controller.js

```diff
@@ -0,0 +1,369 @@
+var app=angular.module("myApp.controllers.aside",['ngRoute','ngAnimate'])
+    .controller("asideCtr", function($scope,$location, $rootScope) {
+        var socket = io.connect();
+        $scope.rightVisible = false;
+        $rootScope.logged= false;
+        $scope.close = function() {
+            if($scope.rightVisible != false)
+            {
+                $scope.rightVisible = false;
+                $rootScope.$broadcast("documentClicked");
+                socket.emit('menuClose',{})
+            }
+        };
+
```

Comment:
I dont see any github authentication. Please authenticate with the GitHub account.